stable/bundle_dep.pony

```diff
@@ -50,3 +52,57 @@ class BundleDepGitHub
     if git_tag isnt None then
       Shell("cd " + root_path() + " && git checkout " + (git_tag as String))
     end
+
+class BundleDepLocalGit
+  let bundle: Bundle
+  let info: JsonObject box
+  let package_name: String
+  let local_path: String
+  let git_tag: (String | None)
+  new create(b: Bundle, i: JsonObject box)? =>
+    bundle       = b
+    info         = i
+    package_name = try info.data("package-name") as String
+                   else bundle.log("No 'package-name' key in dep: " + info.string()); error
+                   end
+    local_path   = try info.data("local-path") as String
+                   else bundle.log("No 'local-path' key in dep: " + info.string()); error
+                   end
+    git_tag =      try info.data("tag") as String
```

Comment:
Don't want to be too annoying, but can you move this `=` to the right to align it with the others?